### ISSUE TITLE
Social sign-in grammar and clarity problems

### ISSUE THREAD

**Issue by jkellyinsf** [aws-amplify/docs#5721]
**Describe the content issue**:

This page needs an overhaul from a capable technical editor.  

1. Grammatical problems appear throughout, for example
- **Common analogies ... is ...**
- **OAuth support ... supports ...**
- **Setup Your Auth Provider**
- **at anytime**

2. The "setup frontend" section (for example) is hard to follow.  
- **"Passing LoginWithAmazon ... will bypass the Hosted UI..."** This is the first mention of Cognito's Hosted UI or "bypassing" it, and the concepts need more introduction.  What is the relationship between Amplify and the Hosted UI?  What does it mean to bypass or not bypass it?
- **"...and federate immediately with the social provider."**  What on earth does this mean?
- **"If you are looking to add a custom state..."** Another reference to a concept that wasn't introduced.  Do you mean "a state" like "pending approval"?  Or "state"  in the React sense of a volatile component property?  Or do you mean an additional column in the cognito user table?

3. Finally, the macro objective is unclear.  If I follow the instructions on the page, what do I end up with?

**URL page where content issue is**:

https://docs.amplify.aws/lib/auth/social/q/platform/js/

**Comment by hibler13** [comment]
Hi @jkellyinsf Thanks for the detailed feedback! We're in the middle of a comprehensive update to this guide and will publish a new version in September. We'll be sure to address the issues you raised in that version. Thanks again!
